src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleCss = {};
+  const changeCss = sourceString.split(';');
+
+  changeCss.forEach((item) => {
+    const [key, value] = item.split(':');
+
+    if (value !== undefined) {
```

Comment:
The check `if (value !== undefined)` is necessary, but it might be more robust to also check if `key` is not an empty string to avoid adding empty keys to the `styleCss` object.